File: MAP/Lab10/ConsoleApp1/ConsoleApp1/Model/state/FileTableData.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text;

namespace ToyInterpreter.Model.state
{
    class FileTableData
    {
        public String filename;
        public StreamReader streamReader;

        public FileTableData(String filename, StreamReader streamReader)
        {
            this.filename = filename;
            this.streamReader = streamReader;
        }

        public FileTableData()
        {
        }
    }
}

File: MAP/PracticalExamPractice/01.CyclicBarrierToyInterpreter/src/model/state/FileTableData.java
package model.state;

import java.io.BufferedReader;

public class FileTableData {
    public String filename;
    public BufferedReader bufferedReader;

    public FileTableData(String filename, BufferedReader bufferedReader) {
        this.filename = filename;
        this.bufferedReader = bufferedReader;
    }

    public FileTableData() {
    }
}
